modules/tinymce/src/plugins/table/main/ts/ui/Buttons.ts

```diff
@@ -194,14 +194,25 @@ const addButtons = (editor: Editor, selectionTargets: SelectionTargets, clipboar
   editor.ui.registry.addMenuButton('tablecellvalign', {
     icon: 'vertical-align',
     tooltip: 'Vertical align',
-    fetch: generateItemsCallback(editor, verticalAlignValues, 'tablecellverticalalign', (item) => item.text, (item) => {
-      editor.execCommand('mceTableApplyCellStyle', false, {
-        'vertical-align': item.value
-      });
-    }),
+    fetch: generateItemsCallback(editor, verticalAlignValues, 'tablecellverticalalign', (item) => item.text, applyTableCellStyle(editor, 'vertical-align')),
```

Comment:
```suggestion
    fetch: generateItemsCallback(
      editor,
      verticalAlignValues,
      'tablecellverticalalign',
      (item) => item.text,
      applyTableCellStyle(editor, 'vertical-align')
    ),
```

These lines are getting quite long now (well over the 160char max we normally try to keep within). Maybe we should "chop" it down like this? I'll add a suggestion for each, but your call 😄